### DISCUSSION TITLE
Pico: SPI

### DISCUSSION THREAD

**Discussion by mkellner** [Moddable-OpenSource/moddable#591] [Porting]
I've added initial SPI support to the Pico. There is also a new target: `pico/ili9341` to support it.
And a wiring guide for the display at: [https://github.com/Moddable-OpenSource/moddable/blob/pico/documentation/displays/wiring-guide-generic-2.4-spi-pico.md](https://github.com/Moddable-OpenSource/moddable/blob/pico/documentation/displays/wiring-guide-generic-2.4-spi-pico.md).

The initial implementation is using SPI1 on the following pins:

```
MISO: 8
MOSI: 11
SCK: 10
```
The ili9341 is connected with the additional pins:
```
CS: 9
DC: 7
RST: 12
```

As the underlying Pico SPI APIs are blocking only at this point, so is the Moddable implementation.

**Comment by wilberforce** [comment]
@mkellner 
Nice work
